mantle/platform/qemu.go

```diff
@@ -747,22 +754,9 @@ func (builder *QemuBuilder) AddDisk(disk *Disk) error {
 
 // AddInstallIso adds an ISO image
 func (builder *QemuBuilder) AddInstallIso(path string, bootindexStr string) error {
```

Comment:
Bikeshed:  this should probably be renamed to `AddIso` now given that (1) the boot index is a freeform argument, which means that it might not even be given priority, and (2) in the `--qemu-image foo.iso` case, it's not really playing the role of an install ISO.

Obviously not a blocker.